src/containers/add-attachments/AddAttachments.tsx

```diff
@@ -0,0 +1,107 @@
+import { ChangeEvent, FC, useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
```

Comment:
move mui imports together with other node modules packages

Reply:
why is this a problem? with spaces it looks more organized and readable

Reply:
cause we collect imports from libraries together to make code clean

Reply:
done